src/app/components/links/basic-link/basic-link.component.ts

```diff
@@ -9,6 +9,12 @@ import {UserResolver} from '../../../resolvers/user/user.resolver';
   styleUrls: ['./basic-link.component.scss'],
   template: `
     <a
+      [angulartics2On]="angulartics2On"
+      [angularticsAction]="angularticsAction"
+      [angularticsCategory]="angularticsCategory"
+      [angularticsLabel]="angularticsLabel"
+      [angularticsProperties]="angularticsProperties"
+      [angularticsValue]="angularticsValue"
```

Comment:
Can we use a directive here? I have no idea what I'm talking about but bear with me.. 
But as I recall directives in new angular versions we could use that to create something like `just-analytics`
which could support all the above ☝️ attributes. See ng-docs @ https://angular.io/guide/attribute-directives. I'm not at all sure if this is how it works, or if its even worth it :) Just a thought and I'm curious about what you think @bex1 :) feel free to ignore this..